graphql/schema/types/repo_settings.graphql

```diff
@@ -7,6 +7,7 @@ update the settings for a given project.
 input RepoSettingsInput {
   repoId: String! @requireProjectAccess(permission: SETTINGS, access: EDIT)
   aliases: [ProjectAliasInput!]
+  githubAppAuth: GithubAppAuthInput @redactSecrets
```

Comment:
Not a change but do you know what the @redactSecrets annotation does out of curiosity?

Reply:
it's for this: https://github.com/evergreen-ci/evergreen/pull/7556/files